Observer/src/observer/Subscriber.java

```diff
@@ -0,0 +1,22 @@
+package observer;
+
+public class Subscriber {
+    private String name;
+    private Channel channel = new Channel();
+
+    Subscriber(String name){
+        this.name = name;
+    }
+
+    public void update() {
+        System.out.println("Video uploaded");
```

Comment:
what message subscriber is receiving here

Reply:
Whenever the video is uploaded, each subscribers will receive the a notification(here just a print statement) about the video uploaded. Like shown below

![image](https://user-images.githubusercontent.com/55446681/201321294-25b61cea-529e-46c9-bc7d-6328b6666a2c.png)